board.py

```diff
@@ -1,11 +1,31 @@
 from pieces import *
+from ascii import *
 
 class Board:
 	def __init__(self):
 		self.board = self
-		self.squares = {}	
+		self.squares = {}
+		self.map = []
+		self.ascii = Ascii()
+		self.white_to_move = True
+		self.algebraic_squares = set()	
 		for key in range (0,64):
 			self.squares[key] = Square(key)
+		columns = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
+		rows = range(8, 0,-1)
+		for column in columns:
+			for row in rows:
+				algebraic_square = column + str(row)
+				self.algebraic_squares.add(algebraic_square)
+
+	def get_map(self):
+		self.map = []
+		for square in self.squares:
+			piece = self.squares[square].piece
+			if piece == None:
+				self.map.append("  ")
+			else:
+				self.map.append(piece.piece_name)
```

Comment:
It's not touched in this PR, but I believe the logic in the method below this, `new_game` has flipped the colors/ranks. Today when I start a new game, the white pieces are shown as on row 8, whereas I believe in traditional notation the white pieces (though at the bottom visually) are considered as on row 1. This, in normal notation, the move `e4` (Pawn from e2 to e4) is a normal first move. Whereas the way you are showing it on the board, and in your logic, that same move is written as `move from e7 to e5`.

Compare your version:
<img width="563" alt="image" src="https://github.com/ryanmjacksonrmj/Chess/assets/8495/ed24ab7d-03a6-43d4-94bd-850f0fb6df6c">

To a standard online game engine:
<img width="628" alt="image" src="https://github.com/ryanmjacksonrmj/Chess/assets/8495/44cb7fe7-2a62-44f8-ad0d-c8f4c8e51f61">

Reply:
I have stared at chess boards over and over again and am unsure how I managed to get the numbers of the rows reversed. Will fix at least on the representation to the user/getting input from the user. Fortunately most of the non-user-facing logic is derived straight from square numbers that the user never sees rather than row numbers.